webapp/src/components/react_select_setting.jsx

```diff
@@ -20,6 +23,58 @@ export default class ReactSelectSetting extends React.PureComponent {
         }
     }
 
+    getStyle = (theme) => ({
```

Comment:
what happens now if theme is undefined or null?

Reply:
@cpoile Looks like previously for some reason, the theme code that I was using to set the style was running at an earlier point where at times the theme was undefined, here however, I do not get the undefined error, so I would assume that the theme would always be there in this case.

Reply:
@cpoile on above

Reply:
@jasonblais we need to put a note for Dylan to test this a lot and be on the lookout for errors right here then, just in case.